README.md

```diff
@@ -0,0 +1,56 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+## Description
+> In this project, I set up a "Hello world" repository. No complex coding is required for this exercise. My goal here is to master all of the tools and best practices I learned about in previous steps. 
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://github.com/DianaBeki/Hello-microverse)
+
+## Getting Started
+
+**If you want to this project kindly check the [ GitHub repository](https://github.com/DianaBeki/Hello-microverse)**
+
+### Prerequisites
+
+-Web Browser
+-Code Editor:[visual Studio code](https://code.visualstudio.com/)
+
+### Setup
+
+```
+git clone git@github.com:DianaBeki/Hello-microverse.git
+```
+
+Open a index.html file with your code editor of your choice and/or a browser of your choice.
+
+```
+Open the project using a live server extension that can be found into your code editor.
+```
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/DianaBeki)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](MIT.md) licensed.
```

Comment:
In the case of README consider not leaving anyyyyyyyyyyyyyyyyyyyyyy of the sections in the [Microverse README Template ](https://github.com/microverseinc/readme-template) template as it's a project requirement.

- The only optional section is FAQ.
- You can delete the live demo section if there is no deployment of the project on some server. In this case, you should delete that section.
- Make sure the to add section Future Features, including 1-3 potential features to be added.